tutorials/rendering/gles2_gles3_differences.rst

```diff
@@ -136,7 +136,10 @@ so if exporting to mobile using GLES2 make sure to set the project setting
 Blend shapes
 ------------
 
-Blend shapes are not supported in GLES2.
+In GLES2, blend shapes are implemented on the CPU instead of the GPU.
+Accordingly, they may not perform as well as Blend Shapes in GLES3. To avoid
```

Comment:
```suggestion
Accordingly, they may not perform as well as blend shapes in GLES3. To avoid
```

This is the only captialized instance, not sure but IMO either captialize all or none.

Reply:
Fixed; I meant to capitalize none of them (it looks a bit off to me).